EdgeCraftRAG/tests/test_pipeline_local_llm.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
We need e2e test pass before merge.

Reply:
Hi suyue ,as we aligned , I will add e2e test later after merge,,I will do it ASAP. thanks